backend/src/packages/orders/libs/exceptions/order-not-found.ts

```diff
@@ -0,0 +1,12 @@
+import { HttpCode, HttpError, HttpMessage } from 'shared/build/index.js';
+
+class OrderNotFound extends HttpError {
```

Comment:
we have already NotFoundError

Reply:
Yes, we have NotFoundError which extends ApplicationError, not HttpError and when throw it returns 500 status, as I throw error with same messsage in several places in order package I added custom error that extends HttpError just for not duplicating code

Reply:
you should try again reuse NotFoundError, it was used for business and it's basic purpose was to create general error for such cases